client/webserver/site/src/css/main.scss

```diff
@@ -71,8 +71,7 @@ input:invalid {
 
 header.maintop {
   width: 100%;
-  height: 70px;
-  min-height: 70px;
+  min-height: 60px;
```

Comment:
It seems that 60px is sufficient for the header section ? I don't see any obvious issues with lowering it to 60 (to save more space) with Brave/Chrome/Firefox.

I'm also not sure why both are defined, isn't it the case that `min-height` always supersedes `height` ? That's [what  I found](https://css-tricks.com/almanac/properties/m/min-height/) when googled.

It's in a separate commit, could be easily removed in case it breaks anything.